File: Codes/uri1048.py
salary = float(input())

if salary>0 and salary<=400:
    salaryf = salary*1.15
    reajuste = salaryf-salary
    percent = 15
elif salary>400 and salary<=800:
    salaryf = salary*1.12
    reajuste = salaryf-salary
    percent = 12
elif salary>800 and salary<=1200:
    salaryf = salary*1.10
    reajuste = salaryf-salary
    percent = 10
elif salary>1200 and salary<=2000:
    salaryf = salary*1.07
    reajuste = salaryf-salary
    percent = 7
elif salary>2000:
    salaryf = salary*1.04
    reajuste = salaryf-salary
    percent = 4
print("Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: {} %".format(salaryf,reajuste,percent))

File: URIonlineJudge/Codes/C/uri1048.c
#include <stdio.h>
int main(){
    float  salario,valor,novo;
    scanf("%f",&salario);
    if(salario>=0 && salario<=400){
        valor=(salario*15)/100.0;
        novo=valor+salario;
        printf("Novo salario: %.2f\n",novo);
        printf("Reajuste ganho: %.2f\n",valor);
        printf("Em percentual: 15 %\n");
    }
    if(salario>=400.01 && salario<=800.00){
        valor=(salario*12)/100.0;
        novo=valor+salario;
         printf("Novo salario: %.2f\n",novo);
        printf("Reajuste ganho: %.2f\n",valor);
        printf("Em percentual: 12 %\n");
    }
    if(salario>=800.01 && salario<=1200.00){
        valor=(salario*10)/100.0;
        novo=valor+salario;
         printf("Novo salario: %.2f\n",novo);
        printf("Reajuste ganho: %.2f\n",valor);
        printf("Em percentual: 10 %\n");
    }
    if(salario>=1200.01 && salario<=2000.00){
        valor=(salario*7)/100.0;
        novo=valor+salario;
         printf("Novo salario: %.2f\n",novo);
        printf("Reajuste ganho: %.2f\n",valor);
        printf("Em percentual: 7 %\n");
    }
    if(salario>2000.00){
        valor=(salario*4)/100.0;
        novo=valor+salario;
         printf("Novo salario: %.2f\n",novo);
        printf("Reajuste ganho: %.2f\n",valor);
        printf("Em percentual: 4 %\n");
    }
    
    return 0;
}
